docs/index.html

```diff
@@ -1,76 +1,116 @@
 	<link rel="mask-icon" href="/icons/safari-pinned-tab.svg" color="#000000">
-	<link rel="shortcut icon" href="/icons/favicon.ico">
-	<meta charset="utf-8">
-	<meta name="color-scheme" content="light dark">
-	<meta name="description" content="">
-	<meta name="msapplication-TileColor" content="#ffffff">
-	<meta name="msapplication-config" content="/icons/browserconfig.xml">
-		<meta name="theme-color" content="#111111" media="(prefers-color-scheme: light)">
-		<meta name="theme-color" content="#eeeeee" media="(prefers-color-scheme: dark)">
-	<meta name="theme-color" content="#ffffff">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-</head>
-<body>
-
-<a href="/">
-	<img alt="MergePHP Logo" src="/logo.jpg">
-</a>
-
-<aside>
-	<ul>
-					<li>Atlanta</li>
-
-					<li>Austin</li>
-
-					<li>Boston</li>
-
-					<li>Kansas City</li>
 
-					<li>Las Vegas</li>
+	<!-- Google Fonts -->
+	<link href="https://fonts.googleapis.com/css2?family=Open+Sans:300;400;600;700;800&display=swap" rel="stylesheet">
 
-					<li>Phoenix</li>
+	<!-- Font Awesome -->
+	<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.0/css/all.min.css" rel="stylesheet">
 
-					<li>San Diego</li>
-
-					<li>Seattle</li>
-
-					<li>Utah</li>
-
-					<li>Vancouver</li>
-
-			</ul>
-</aside>
-	<section>
-		Join us on
-		<time datetime="2023-06-08T19:00:00-04:00" style="display:inline">June 8th, 2023</time>
-		as Israel Carberry presents
-		<a href="/meetups/2023/06/08/symfony-to-laravel-lessons-learned.html"><strong>Symfony to Laravel: Lessons Learned</strong></a>
-		<br>
-		<p>Symfony and Laravel have emerged as the leading MVC frameworks for rapidly building and deploying web applications using PHP. While each has its advocates and naysayers, let's compare them in a way that highlights the strengths of each and their approaches to aiding the delivery of value. Whether you're a tech lead wanting to better guide the maturity of your systems, an engineer looking to broaden your knowledge, or just wanting to know if PHP is a viable option as a component in your stack, awareness of these differences can help smooth the path ahead of you.</p>
-
-		<br>
-					RSVP:
-							<a href="https://www.meetup.com/austinphp/events/292881946/">Austin</a>, 
-							<a href="https://www.meetup.com/seaphp/events/293618983/">Seattle</a>
-						</section>
-
-<a href="/meetups/2023/">View past meetups</a>
-
-<footer>
-	&copy; MergePHP 2023. Source code available <a href="https://www.github.com/mergephp/mergephp-website/">on GitHub</a>
-</footer>
+	<!-- Customized Bootstrap Stylesheet -->
+	<link href="/css/style.css" rel="stylesheet">
+</head>
 
+<body>
+<div class="wrapper">
+	<div class="sidebar">
+		<div class="sidebar-text d-flex flex-column h-100 justify-content-center text-center">
+			<a href="/"><img class="mx-auto d-block bg-primary img-fluid mb-4" alt="MergePHP Logo" src="/images/logo.jpg"></a>
```

Comment:
What are you treating as the root/index of the site? And how are you testing the site?

The reason I ask is I opened `/docs/index.html` on my local machine and the logo image is broken. I was able to fix it by changing `/images/logo.jpg` to `images/logo.jpg`. I had the same issue with the link to `/css/styles.css` above. I had to remove the leading slash to get it to work.

Reply:
@BBBThunda how are you serving the site?  I used `composer serve` and the images show up as expected

Reply:
@cookieguru I tried it with `composer serve` and now it looks good. Thanks!